### ISSUE TITLE
wails3 generate bindings are invalid

### ISSUE THREAD

**Issue by hq-zhonger** [wailsapp/wails#3061]
### Description

wails3 generate bindings No error is reported and no file is generated

### To Reproduce

wails3 generate bindings

### Expected behaviour

wails3 generate bindings自动生成go方法js绑定

### Screenshots

<img width="974" alt="屏幕截图 2023-11-17 224227" src="https://github.com/wailsapp/wails/assets/78897251/fdddda13-f27a-4b51-a05f-67875f252cf6">

### System Details

```shell
Wails Doctor          
                                


# Build Environment
Wails CLI    | v3.0.0-alpha.2
Go Version   | go1.21.4
-buildmode   | exe
-compiler    | gc
CGO_CFLAGS   |
CGO_CPPFLAGS |
CGO_CXXFLAGS |
CGO_ENABLED  | 1
CGO_LDFLAGS  |
GPU 1             | AMD Radeon(TM) Graphics (Advanced Micro Devices, Inc.) - Driver: 27.20.14028.14001
Memory            | 16GB
```

**Comment by ivila** [comment]
@hq-zhonger have you add the json tag in your model?

**Comment by hq-zhonger** [comment]
yes,i use the json tag.This project is a v2 upgrade to v3, with wails3 init -t vue set a new template


陈灏清
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "wailsapp/wails"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月20日(星期一) 上午9:42
***@***.***&gt;;
抄送:&nbsp;"chen hao ***@***.******@***.***&gt;;
主题:&nbsp;Re: [wailsapp/wails] wails3 generate bindings are invalid (Issue #3061)





 
@hq-zhonger have you add the json tag in your model?

**Comment by leaanthony** [comment]
Thanks for raising this, however we have a strict v3 feedback procedure outlined here: https://v3alpha.wails.io/getting-started/feedback/

**Comment by leaanthony** [comment]
Thanks for the report. This was on `v3.0.0-alpha.2` (Nov 2023), and the v3 binding generator has been substantially rewritten since — it's now a type-checker-based generator (`golang.org/x/tools/go/packages`) with a large test suite and proper error/warning reporting, so it no longer silently produces nothing.

Given how far the generator has moved on, I'm closing this as stale. If you can still reproduce 'no files generated' on a current v3 release, please open a fresh issue with a minimal repro and your `wails3 doctor` output. Thanks!
